macros/edr/tests/test_utils/get_anomaly_query.sql

```diff
@@ -51,12 +54,25 @@
     {{- return(anomaly_query) -}}
 {% endmacro %}
 
-{%- macro is_score_anomalous_condition(sensitivity) -%}
-    {%- set spikes_only_metrics = ['freshness', 'event_freshness'] -%}
+{%- macro set_directional_anomaly(anomaly_direction, anomaly_score, sensitivity) -%}
```

Comment:
Just a thought as I see text comparison all over the code - 
Would this config work? Should it?
```
      - elementary.volume_anomalies:
          anomaly_direction: "Drop"
```

Reply:
You're right, I changed it now to use "ilike" instead

Reply:
Maybe we should do something like what @RoiTabach did here:
https://github.com/elementary-data/dbt-data-reliability/pull/310/files/d304e37245216ecd1a781faf4346ff0e66871b6d#diff-3d90a1c0febaf4a23f00626f6686b1f1f2c17d1bd5f4f5f1e93b23f2ed0c91d6:~:text=macros/edr/tests/test_utils/validate_test_parameters.sql

What do you think?

Reply:
Nice! 
Though I'm not sure having case-sensitivity checks is user friendly enough, for now I added a validation to only test if the parameters has the right string and I accept lower/higher cases, if you think it's redundant and we should just fail on upper case parameters let me know and I'll change it :)  
@Maayan-s